api/v1/inference_server.proto

```diff
@@ -22,10 +29,21 @@ message CreateChatCompletionRequest {
     }
 
     // The type of content varies. For eample, it is string or array for UserMesasge.
+    // Refer to https://platform.openai.com/docs/api-reference/chat/create#chat-create-messages
+    message Content {
+      ContentType type = 1;
+      string text = 2;
+      string image_url = 3;
+      string input_audio = 4;
```

Comment:
This should be a proto message that contains `data` and `format`?

<img width="644" alt="image" src="https://github.com/user-attachments/assets/00c9e991-9333-4eaa-aa87-41773cbc969d">